Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM ubuntu
```

Comment:
production images should use a pinned version of base images to ensure long term repeatability

Reply:
Do you recommend a version?

For example, it's recommended to use older distros when building CPython for libc compatibility. Are there any such caveats in micropython?

Reply:
The older versions for cpython are due to compatibility with dynamically linked libraries. Micropython doesn't dynamically link to anything external, so no there isn't any restriction there.

The compiler version is the most important thing, and with gcc newer is often better. The esp ones have their own version pin.

As far as the distro/image pin the current released version should be fine, it's more about ensuring that when there's a new version of the image released upstream, it isn't automatically used without explicit testing for compatibility.